jumpscale/core/base/fields.py

```diff
@@ -262,12 +260,33 @@ def validate(self, value):
         Returns:
             Boolean expresion"""
         super().validate(value)
-        default_format = "%H:%M"
         try:
-            time.strptime(value, default_format)
+            time.strptime(value, self.default_format)
         except Exception:
             raise ValidationError(f"{value} is not a valid Time")
 
+    def add_hours(self, hours):
```

Comment:
Where thes add_hours, subtract_hours, _hours_operations are used?

Reply:
the add_hours and subtact_hours are exposed to the user to add or subtract the desired number of hours